account/src/main/java/com/user/account/advice/CustomExceptions.java

```diff
@@ -0,0 +1,9 @@
+package com.user.account.advice;
+
+
+public class CustomExceptions extends Exception{
```

Comment:
Extend `RuntTimeException`, we can discuss if you should have Checked or unchecked

But here i think it should be unchecked.

Reply:
Extended custom exception with run time exception, Since haven't checked for the condition that provided account number exist or not.